src/script/descriptor.cpp

```diff
@@ -1081,7 +1090,10 @@ std::unique_ptr<DescriptorImpl> ParseScript(uint32_t& key_exp_index, Span<const
             }
             auto arg = Expr(expr);
             auto pk = ParsePubkey(key_exp_index, arg, ctx, out, error);
-            if (!pk) return nullptr;
+            if (!pk) {
+                error = strprintf("Multi: %s", error);
```

Comment:
I don't understand why this can't (or shouldn't) be more specific?

Reply:
I think it's just that it would result in a fairly large `if` block to get the right function name when `Multi:` is sufficient to get the point across. There can only be one multi in a descriptor anyways, so it should be obvious where the error is.

Reply:
```suggestion
                error = strprintf("%smulti: %s", sorted_multi ? "sorted" : "", error);
```

?